cli/cmd/rm.go

```diff
@@ -21,6 +21,6 @@ Any data which is not in a volume will be lost.`,
 func init() {
 	RootCmd.AddCommand(rmCmd)
 	rmCmd.SetUsageTemplate(subCommandTemplate)
-	rmCmd.Flags().BoolP("force", "f", false, "Don't ask to confirm removal")
```

Comment:
That worries me a bit... if it's a limitation of `cobra` we are in a *pickle*, because it will break compatibility with `docker-compose`…